DuckDuckGo/NetworkProtectionTunnelController.swift

```diff
@@ -24,6 +24,7 @@ import Combine
 import Core
 import NetworkExtension
 import NetworkProtection
+import Common
```

Comment:
Is this required? (I had to do the same thing with a few macOS classes lately, so this is a genuine question 😄)

Reply:
Good catch, in this instance it’s not necessary at all :D